### DISCUSSION TITLE
How Do I Tell Which Version of Sloeber I have?

### DISCUSSION THREAD

**Discussion by gfvalvo** [Sloeber/arduino-eclipse-plugin#1667] [Q&A]
Hi All.

So, with most Windows programs you can click Help --> About and you'll see a simple and clear display that includes the version number you're running. Not so with Sloeber. Instead you get a window with multiple options. Clicking any of those brings up a confusing display of unhelpful information. So, is there a simple way to determine that I'm running Version 4.4.x?

![image](https://github.com/Sloeber/arduino-eclipse-plugin/assets/23526904/1c7fdca1-3e0b-4afb-875d-7ae25b3ff00d)

**Comment by jantje** [comment]
Click the installation details and scroll down
![image](https://github.com/Sloeber/arduino-eclipse-plugin/assets/1175841/555ccfcb-88c7-4d74-aafd-f898d257403a)
Because eclipse is a bundle of plugins (each with their version number) there is not really a Sloeber version number.

The same issue would exists if you would try to give your pc software as a whole a version number.
